plugin/src/sbt-test/scalafmt-sbt/sbt/test

```diff
@@ -74,24 +74,6 @@ $ copy-file p13/src/test/scala/Test3.scala p13/src/main/scala/Test3.scala
 $ touch p13/src/main/scala/Test.scala
 -> p13/scalafmt
 
-> p14/scalafmtCheck
-# prevent reading the source (without changing the mtime) to detect actual/uncached invocation of scalafmt
-$ exec chmod 000 p14/src/main/scala/Test.scala
-######## no-op expected when nothing has changed
-> p14/scalafmtCheck
-> p14/scalafmt
```

Comment:
Removed p14 because scalafmt here should not be a no-op just because scalafmtCheck moved the tracker forward.

Reply:
PR commits were squashed when merging so this might not be obvious, but [this was protecting against regressions](https://github.com/scalameta/sbt-scalafmt/commit/e2a0d59546a7d4b23ba41e03caad9efcfcf8fb94) of #39, and should remain relevant despite the implementation change

Reply:
ok. I think I understand now. In case `scalfmtCheck` succeeded, I should stop passing `pending` file change since there's nothing to (re)format?

Reply:
@poslegm Looking at your error message, it makes sense what's happening. I'll fix to address that.